config/blah.config.template

```diff
@@ -80,7 +80,7 @@ blah_require_proxy_on_submit=
 
 #Path where PBS executables are located 
 # NOTE: this path is computed many times; I worry about the overhead here.  -BB, 12-13-2012
-pbs_binpath=`which qsub 2>/dev/null|sed 's|/[^/]*$||'`
+pbs_binpath=/usr/bin
```

Comment:
And it modifies it by default so the only cases we have to worry about our people using our `blahp` but not using `osg-configure` and the default insane bin paths.